app_feup/lib/controller/logout.dart

```diff
@@ -21,7 +21,7 @@ Future logout(BuildContext context) async {
   final prefs = await SharedPreferences.getInstance();
   await prefs.clear();
 
-  AppLecturesDatabase().deleteLectures();
+  AppHtmlLecturesDatabase().deleteLectures();
```

Comment:
Well the problem is more the schema which we were using to store the lectures. It's in no way uniform and converting what comes from the website itself to what is being expected there is less than ideal.

Reply:
I think this schema is more concise, but perhaps we could try and use this opportunity to reach a consensus schema that's not as tightly connected to the constructors.

Reply:
The only difference is the start and end times' types, right? Maybe change them do DateTime to be consistent with the exams too

Reply:
I've then since noticed that the start time from seconds to String had to be calculated nonetheless when using data from the API, so I'm just going to update it to use startTime since that's standardized already between the two places.